nixos/modules/services/cluster/kubernetes/default.nix

```diff
@@ -222,14 +245,27 @@ in {
     })
 
     (mkIf cfg.kubelet.enable {
-      virtualisation.docker = {
-        enable = mkDefault true;
-
-        # kubernetes needs access to logs
-        logDriver = mkDefault "json-file";
-
-        # iptables must be disabled for kubernetes
-        extraOptions = "--iptables=false --ip-masq=false";
+      systemd.services.containerd =
```

Comment:
I think creating a `containerd` module would be better rather than adding it here, would also allow it to be used independent of k8s.

Reply:
Will look into separating this now.

Reply:
First version separate module here (7fc7d420b32f8dae291990a8ef5f6670a98f43d4)